site/content/integrate/apps/structure/manifest/_index.md

```diff
@@ -57,40 +64,40 @@ An example manifest looks like this:
 
 ### Permissions
 
-Apps need to request permissions from the admin who is installing them. After an app is installed and available to end users, each user in Mattermost will need to "authenticate" with your app. Using the `act_as_user` permission, your app will be performing an API call on each authenticated user's behalf.
-
-Below are the permissions ([godoc](https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps#Permission)) an app can currently ask for:
+Apps need to request permissions from the System Admin who is installing them. After an App is installed and available to end users, each user in Mattermost will need to "authenticate" with your App.
+Using the `act_as_user` permission, your App will be performing an API call on behalf of each authenticated user.
 
-| Name                               | Description                                                                                                                                         |
-|:-----------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------|
-| `user_joined_channel_notification` | Be notified when users join channels.                                                                                                               |
-| `act_as_bot`                       | Use Mattermost REST API as the app's bot user.                                                                                                      |
-| `act_as_user`                      | Use Mattermost REST API as connected users.                                                                                                         |
-| `remote_oauth2`                    | Use remote (third-party) OAuth2 support, and will store secrets and tokens to third-party system(s) within the Mattermost server.                   |
-| `remote_webhooks`                  | Receive webhooks from a remote (third-party) system, and process them as a bot (which will typically, will add a notification message to a channel) |
+The following table lists the supported permissions ({{<newtabref title="godoc" href="https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps#Permission">}}) an app can request:
 
+| Name                               | Description                                                                                                                                 |
+|:-----------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------|
+| `user_joined_channel_notification` | Be notified when users join channels.                                                                                                       |
```

Comment:
This one (`user_joined_channel_notification`) will be gone very soon, I recommend removing now

Reply:
Is there a known release when this notification type will be removed?
Is it already in a deprecated state?

Reply:
yes (deprecated), it really was an accident to have it to begin with, was meant to be an example to expand on later... Please remove.